packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -86,8 +86,8 @@ const COLUMN_STATE = {
 const COLUMN_DISK = {
   itemRenderer: remote =>
     remote.info !== undefined &&
-    remote.info.used !== undefined &&
-    remote.info.size !== undefined && (
+    remote.info.used != null &&
```

Comment:
Isn't it better to use `!==` ?

Reply:
I don't know if they can be `undefined`.